tools/templates/vfvs_run.py

```diff
@@ -784,16 +1262,17 @@ def process(ctx):
             # Check to see if ligand contains B, Si, Sn or has duplicate coordinates
             with open(ligand['path'], "r") as read_file:
                 for index, line in enumerate(read_file):
-
-                    match = re.search(r'(?P<letters>\s+(B|Si|Sn)\s+)', line)
-                    if(match):
-                        matches = match.groupdict()
-                        logging.error(
-                            f"Found {matches['letters']} in {collection_full_name}/{ligand_key}. Skipping.")
-                        skip_reason = f"failed(ligand_elements:{matches['letters']})"
-                        skip_reason_json = f"ligand includes elements: {matches['letters']})"
-                        skip_ligand = 1
-                        break
+                    
+                    if ctx['config']['run_atom_check']: 
```

Comment:
This needs to be added to the all.ctrl file

Reply:
Good point! I forgot to push the updated all.ctrl file. Will update